pkg/apis/resource/types.go

```diff
@@ -97,9 +97,9 @@ type ResourceClaimStatus struct {
 	// +optional
 	DriverName string
 
-	// Allocation is set by the resource driver once a resource has been
-	// allocated successfully. If this is not specified, the resource is
-	// not yet allocated.
+	// Allocation is set by the resource driver once a (set of) resources has
```

Comment:
nit for followup because TZ:  "once a resource or set of resources has been"

Reply:
Updated